WordPress/Classes/ViewRelated/Reader/Tags View/ReaderTopicCollectionViewCoordinator.swift

```diff
@@ -40,45 +40,45 @@ class ReaderTopicCollectionViewCoordinator: NSObject {
 
     weak var delegate: ReaderTopicCollectionViewCoordinatorDelegate?
 
-    let collectionView: UICollectionView
+    weak var collectionView: UICollectionView?
```

Comment:
Nitpick, just to visually separate the two:

```suggestion
    weak var collectionView: UICollectionView?

```